subt/artf_node.py

```diff
@@ -84,13 +84,13 @@ def __init__(self, config, bus):
         self.fx = config.get('fx', 554.25469)  # use drone X4 for testing (TODO update all configs!)
 
     def create_detector(self):
-        model = os.path.join(os.path.dirname(__file__), '../../../mdnet1.64.64.13.4.relu.pth')
+        model = os.path.join(os.path.dirname(__file__), '../../../mdnet3.64.64.13.4.relu.pth')
         confidence_thresholds = {
             'survivor': 0.8,
-            'backpack': 0.95,
-            'phone': 0.77,
-            'helmet': 0.9,
-            'rope': 0.8
+            'backpack': 0.8,
+            'phone': 0.5,
+            'helmet': 0.5,
+            'rope': 0.6
```

Comment:
Similar question as for the phone. The thresholds of both detectors are really low, so I would like to understand why is it ok/safe/good idea.

Reply:
For helmet, there is significant amount of the positive detection between 0.5 and 0.7. Yes, this value basically turn off the mdnet.

For rope, it the same as for phone. There are not false positive detection.